From c52e03e621e940ba99b3f7a8747b56ed8c7d704f Mon Sep 17 00:00:00 2001
From: Nikita Chernikov <nchernikov@maxlinear.com>
Date: Tue, 3 Mar 2026 14:39:30 +0200
Subject: [PATCH] WLANRTSYS-95669 Remove accidentally added patch files to repo
 introduced in Triband MLD commit 649be78a9d2

Fixup to: WLANRTSYS-96596 MLO: Triband MLD Support

Signed-off-by: Nikita Chernikov <nchernikov@maxlinear.com>
---
 0001-WLANRTSYS-96596-CR-Fixes-1.patch      | 125 ---------------------
 0001-WLANRTSYS-96596-Fix-indentation.patch |  42 -------
 2 files changed, 167 deletions(-)
 delete mode 100644 0001-WLANRTSYS-96596-CR-Fixes-1.patch
 delete mode 100644 0001-WLANRTSYS-96596-Fix-indentation.patch

diff --git a/0001-WLANRTSYS-96596-CR-Fixes-1.patch b/0001-WLANRTSYS-96596-CR-Fixes-1.patch
deleted file mode 100644
index ce68b6903..000000000
--- a/0001-WLANRTSYS-96596-CR-Fixes-1.patch
+++ /dev/null
@@ -1,125 +0,0 @@
-From 1581f52898870f26c6a106b6b31cfb609bad5f94 Mon Sep 17 00:00:00 2001
-From: Nikita Chernikov <nchernikov@maxlinear.com>
-Date: Sun, 1 Feb 2026 12:26:28 +0200
-Subject: [PATCH] WLANRTSYS-96596 CR Fixes 1
-
-Signed-off-by: Nikita Chernikov <nchernikov@maxlinear.com>
----
- src/ap/ctrl_iface_ap.c |  2 +-
- src/ap/drv_callbacks.c |  8 ++++----
- src/ap/hostapd.c       |  2 ++
- src/ap/ieee802_11.c    | 16 ++++++++--------
- 4 files changed, 15 insertions(+), 13 deletions(-)
-
-diff --git a/src/ap/ctrl_iface_ap.c b/src/ap/ctrl_iface_ap.c
-index 4f920d9d0..922c6dd5a 100644
---- a/src/ap/ctrl_iface_ap.c
-+++ b/src/ap/ctrl_iface_ap.c
-@@ -1147,11 +1147,11 @@ int hostapd_ctrl_iface_status(struct hostapd_data *hapd, char *buf,
- 
- #ifdef CONFIG_IEEE80211BE_MXL_MLO
- 	if (mxl_is_mlo_enabled(hapd->conf)) {
--		struct hostapd_data *sib_bss;
- 		struct p_ap_mld *mld = mxl_ml_get_ap_mld(hapd);
- 		int idx = 0;
- 
- 		if (mld) {
-+			struct hostapd_data *sib_bss;
- 			ret = os_snprintf(buf + len, buflen - len,
- 					  "mld_id=%d\n"
- 					  "mld_addr=" MACSTR "\n"
-diff --git a/src/ap/drv_callbacks.c b/src/ap/drv_callbacks.c
-index 7763c9cc9..0b1ef15f9 100644
---- a/src/ap/drv_callbacks.c
-+++ b/src/ap/drv_callbacks.c
-@@ -592,16 +592,16 @@ int hostapd_notif_assoc(struct hostapd_data *hapd, const u8 *addr,
- 			sta->wpa_sm = wpa_auth_sta_init(hapd->wpa_auth,
- 							sta->addr,
- 							p2p_dev_addr);
--#ifdef CONFIG_IEEE80211BE_MXL_MLO
--		if (sta->wpa_sm && mxl_is_mlo_enabled(hapd->conf) && mxl_mlo_is_ml_sta(sta))
--			mxl_wpa_mld_auth_sta_init(sta);
--#endif /* CONFIG_IEEE80211BE_MXL_MLO */
- 
- 		if (sta->wpa_sm == NULL) {
- 			wpa_printf(MSG_ERROR,
- 				   "Failed to initialize WPA state machine");
- 			return -1;
- 		}
-+#ifdef CONFIG_IEEE80211BE_MXL_MLO
-+		if (mxl_is_mlo_enabled(hapd->conf) && mxl_mlo_is_ml_sta(sta))
-+			mxl_wpa_mld_auth_sta_init(sta);
-+#endif /* CONFIG_IEEE80211BE_MXL_MLO */
- 		wpa_auth_set_rsn_selection(sta->wpa_sm, elems.rsn_selection,
- 					   elems.rsn_selection_len);
- #ifdef CONFIG_IEEE80211BE
-diff --git a/src/ap/hostapd.c b/src/ap/hostapd.c
-index 5ce1c9bcf..59d96f852 100644
---- a/src/ap/hostapd.c
-+++ b/src/ap/hostapd.c
-@@ -5162,6 +5162,7 @@ static int hostapd_reconf_add_bss(struct hostapd_iface *iface, struct hostapd_co
- 		return -1;
- 	}
- 
-+#ifdef CONFIG_IEEE80211BE_MXL_MLO
- 	/* Reload all non-TX BSS in MBSSID since wpa auth of TX BSS has changed */
- 	if (mxl_hostapd_is_mbssid_tx_bss(iface->bss[idx])) {
- 		for (size_t j = MULTIBSS_REFERENCE_BSS_IDX; j < iface->num_bss; j++) {
-@@ -5171,6 +5172,7 @@ static int hostapd_reconf_add_bss(struct hostapd_iface *iface, struct hostapd_co
- 			hostapd_reload_bss_only(iface->bss[j]);
- 		}
- 	}
-+#endif /* CONFIG_IEEE80211BE_MXL_MLO */
- 
- 	return 0;
- }
-diff --git a/src/ap/ieee802_11.c b/src/ap/ieee802_11.c
-index b1c012b73..4925a29a8 100644
---- a/src/ap/ieee802_11.c
-+++ b/src/ap/ieee802_11.c
-@@ -4390,10 +4390,6 @@ u16 owe_process_rsn_ie(struct hostapd_data *hapd,
- 	if (!sta->wpa_sm)
- 		sta->wpa_sm = wpa_auth_sta_init(hapd->wpa_auth,	sta->addr,
- 						NULL);
--#ifdef CONFIG_IEEE80211BE_MXL_MLO
--	if (sta->wpa_sm && mxl_is_mlo_enabled(hapd->conf) && mxl_mlo_is_ml_sta(sta))
--		mxl_wpa_mld_auth_sta_init(sta);
--#endif /* CONFIG_IEEE80211BE_MXL_MLO */
- 
- 	if (!sta->wpa_sm) {
- 		wpa_printf(MSG_WARNING,
-@@ -4401,6 +4397,10 @@ u16 owe_process_rsn_ie(struct hostapd_data *hapd,
- 		status = WLAN_STATUS_UNSPECIFIED_FAILURE;
- 		goto end;
- 	}
-+#ifdef CONFIG_IEEE80211BE_MXL_MLO
-+	if (mxl_is_mlo_enabled(hapd->conf) && mxl_mlo_is_ml_sta(sta))
-+		mxl_wpa_mld_auth_sta_init(sta);
-+#endif /* CONFIG_IEEE80211BE_MXL_MLO */
- #ifdef CONFIG_IEEE80211BE
- 	if (ap_sta_is_mld(hapd, sta))
- 		wpa_auth_set_ml_info(sta->wpa_sm,
-@@ -4822,16 +4822,16 @@ static int __check_assoc_ies(struct hostapd_data *hapd, struct sta_info *sta,
- 			sta->wpa_sm = wpa_auth_sta_init(hapd->wpa_auth,
- 							sta->addr,
- 							p2p_dev_addr);
--#ifdef CONFIG_IEEE80211BE_MXL_MLO
--			if (sta->wpa_sm && mxl_is_mlo_enabled(hapd->conf) && mxl_mlo_is_ml_sta(sta))
--				mxl_wpa_mld_auth_sta_init(sta);
--#endif /* CONFIG_IEEE80211BE_MXL_MLO */
- 
- 			if (!sta->wpa_sm) {
- 				wpa_printf(MSG_WARNING,
- 					   "Failed to initialize RSN state machine");
- 				return WLAN_STATUS_UNSPECIFIED_FAILURE;
- 			}
-+#ifdef CONFIG_IEEE80211BE_MXL_MLO
-+			if (mxl_is_mlo_enabled(hapd->conf) && mxl_mlo_is_ml_sta(sta))
-+				mxl_wpa_mld_auth_sta_init(sta);
-+#endif /* CONFIG_IEEE80211BE_MXL_MLO */
- 		}
- 
- #ifdef CONFIG_IEEE80211BE
--- 
-2.47.0.windows.2
-
diff --git a/0001-WLANRTSYS-96596-Fix-indentation.patch b/0001-WLANRTSYS-96596-Fix-indentation.patch
deleted file mode 100644
index 502c427bb..000000000
--- a/0001-WLANRTSYS-96596-Fix-indentation.patch
+++ /dev/null
@@ -1,42 +0,0 @@
-From 7db137ffb28eb3eda083fef93adf06c549de80bf Mon Sep 17 00:00:00 2001
-From: Nikita Chernikov <nchernikov@maxlinear.com>
-Date: Mon, 2 Feb 2026 12:52:53 +0200
-Subject: [PATCH] WLANRTSYS-96596 Fix indentation
-
-Signed-off-by: Nikita Chernikov <nchernikov@maxlinear.com>
----
- src/ap/ctrl_iface_ap.c | 14 +++++++-------
- 1 file changed, 7 insertions(+), 7 deletions(-)
-
-diff --git a/src/ap/ctrl_iface_ap.c b/src/ap/ctrl_iface_ap.c
-index 922c6dd5a..ef3bf74a9 100644
---- a/src/ap/ctrl_iface_ap.c
-+++ b/src/ap/ctrl_iface_ap.c
-@@ -772,17 +772,17 @@ int hostapd_ctrl_iface_deauthenticate(struct hostapd_data *hapd,
- 		mxl_hostapd_send_protected_deauth(hapd, reason);
- 		hostapd_free_stas(hapd);
-    } else {
--      hostapd_drv_sta_deauth(hapd, addr, reason);
-+		hostapd_drv_sta_deauth(hapd, addr, reason);
- #ifdef CONFIG_IEEE80211BE_MXL_MLO
- 		if (sta)
--			mxl_ap_sta_deauth(hapd, sta, reason);
-+		mxl_ap_sta_deauth(hapd, sta, reason);
- #else /* CONFIG_IEEE80211BE_MXL_MLO */
--      if (sta)
--         ap_sta_deauthenticate(hapd, sta, reason);
-+		if (sta)
-+			ap_sta_deauthenticate(hapd, sta, reason);
- #endif /* CONFIG_IEEE80211BE_MXL_MLO */
--      else if (addr[0] == 0xff)
--         hostapd_free_stas(hapd);
--   }
-+		else if (addr[0] == 0xff)
-+			hostapd_free_stas(hapd);
-+	}
- #else /* CONFIG_VENDOR_MXL */
- 	sta = ap_get_sta(hapd, addr);
- 	if (os_strstr(txtaddr, " tx=0")) {
--- 
-2.47.0.windows.2
-
-- 
2.43.0

